### PR TITLE
[regression-test](join)Add test case of left join sql

### PR DESCRIPTION
**Pull request by smallhibiscus**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Add test case of left join sql.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: regression-test/suites/query/join/test_left_join.groovy [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_left_join", "query,p0") {
+    def DBname = "test_left_join"
+    sql "DROP DATABASE IF EXISTS ${DBname}"
```
Comment (dataroaring) [inline regression-test/suites/query/join/test_left_join.groovy:20]: we'd better not create database, just use it. and the case can be written in sql.

Comment (smallhibiscus) [reply regression-test/suites/query/join/test_left_join.groovy]: Thanks，I have modified the case in sql.

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
mv it to query_p0 pls.

**Review by dataroaring — CHANGES_REQUESTED** [review]
move it to query_p0/

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request closed without merging**